zigzag/api.py

```diff
@@ -231,6 +231,84 @@ def get_hardware_performance_zigzag_without_unused_memory(
 
     return cmes[0][0].energy_total, cmes[0][0].latency_total2, cmes
 
+def get_hardware_performance_zigzag_with_mix_spatial_mapping(
```

Comment:
How is this API call different from the other ones? If there are no differences we should avoid making new api calls

Reply:
I suppose only the last two parameters given to the MainStage are new. This api call is fine then.

Reply:
Yes, that's correct. Only the last two parameters are different. All the others are the same to function `get_hardware_performance_zigzag_without_unused_memory`.